src/index.js

```diff
@@ -108,6 +185,9 @@ export default function transformCssModules({ types: t }) {
                     }
                 });
 
+                // wrap or define processCss function that collect generated css
+                currentConfig.processCss = pushStylesCreator(currentConfig.processCss);
```

Comment:
Are we sure that `processCss` is not transforming css in some way and returns it? Because if so then we need to update `pushStylesCreator` to reflect those changes before css is passed to `extractCssFile`.